### PR TITLE
[AlertDialog] Add chromatic story

### PR DESCRIPTION
**Pull request by benoitgrelard**

### CODE REVIEW

File: packages/react/alert-dialog/src/AlertDialog.stories.tsx [hunk 1]
```diff
@@ -48,24 +48,121 @@ export const Controlled = () => {
...
+          <AlertDialogAction className={actionAttrClass}>Confirm</AlertDialogAction>
+          <AlertDialogCancel className={cancelAttrClass}>Cancel</AlertDialogCancel>
+        </AlertDialogContent>
+      </AlertDialog>
+    </div>
+  </div>
+);
+Chromatic.parameters = { chromatic: { disable: false } };
+
+const triggerClass = css({});
```
Comment (jjenzz) [inline packages/react/alert-dialog/src/AlertDialog.stories.tsx:165]: Doesn't look like we need this? But maybe you did it for consistency?

File: packages/react/alert-dialog/src/AlertDialog.stories.tsx [hunk 2]
```diff
@@ -103,21 +200,45 @@ const StyledContent = styled('div', {
...
   border: 'none',
   backgroundColor: '$red',
   color: '$white',
 });
 
-const StyledTitle = styled('h2', {});
+const titleClass = css({});
 
-const StyledDescription = styled('p', {});
+const descriptionClass = css({});
```
Comment (jjenzz) [inline packages/react/alert-dialog/src/AlertDialog.stories.tsx:220]: Same question here, looks like you purposefully left them for some reason?

**Review by jjenzz — APPROVED** [review]
couple questions but otherwise :+1:

**Comment by jjenzz** [comment]
One other thing, I was playing with the canvas view here: https://www.chromatic.com/component?appId=601857eb614bae0021c9b9dd&name=Components%2FAlertDialog&mode=interactive&buildNumber=50

I noticed that when I click cancel on the Uncontrolled dialog, it also closes the data attribute dialog and I can never re-open it after that. Do we have some re-used `id`s somewhere?

File: packages/react/alert-dialog/src/AlertDialog.stories.tsx (hunk 1 above)
Comment (benoitgrelard) [reply packages/react/alert-dialog/src/AlertDialog.stories.tsx:165]: Yeah we do this everywhere for consistency.

**Comment by benoitgrelard** [comment]
> One other thing, I was playing with the canvas view here: https://www.chromatic.com/component?appId=601857eb614bae0021c9b9dd&name=Components%2FAlertDialog&mode=interactive&buildNumber=50
> 
> I noticed that when I click cancel on the Uncontrolled dialog, it also closes the data attribute dialog and I can never re-open them after that. Do we have some re-used `id`s somewhere?

No it's because there are multiple open but not on top of each other and then the controlled one has no `onOpenChange` so can't be closed. These aren't meant to be interactive really.

**Comment by jjenzz** [comment]
Cool, feel free to push the button 🎉

**Pull request merged**